.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+## JIRA Ticket
+
+[<JIRA_TICKET>](https://jira.csiro.com/browse/<JIRA_TICKET>)
+
+## Description
+
+Briefly describe the purpose of this pull request.
+
+## Proposed Changes
+
+List the changes made in this PR.
+
+## How to Test
+
+Explain how to test the changes.
+
+## Additional Information
+
+Add any additional information or context about the pull request here.
+
+## Checklist
+
+- [ ] I have confirmed all commits have been signed.
+- [ ] I have added JSDoc style comments to any new functions or classes.
```

Comment:
Could we add something about documentation e.g. README(s) updated, developer process guides updated etc. Maybe

"Relevant documentation such as READMEs, guides, and class comments are updated?"

Reply:
Actioned